spec/features/breadcrumbs_spec.rb

```diff
@@ -51,7 +51,7 @@
       end
     end
 
-    context 'when on case details page' do
+    context 'when on case details page', :stub_defendants_case_search do
```

Comment:
before, the breadcrumbs, was tested against V1 API endpoint.
Not it uses the V2, which needs `stub_defendants_case_search`